Doc: Add more hyperlinks within the included documentation (#11288)
OpenTTD/OpenTTD: README.md

- the OpenTTD wiki has a [page listing OpenTTD communities](https://wiki.openttd.org/en/Community/Community) including some in languages other than English

### 2.1) Multiplayer games

> You can play OpenTTD with others, both as a competitive game, and as a cooperative game.

Reviewer: ```suggestion
You can play OpenTTD with others, either cooperatively or competitively.
```
I think this is an `either`, not a `both`.

Author: Doing both at the same time is tricky, yes :P
